src/pages/editor/PublishingStrategyOverview.tsx

```diff
@@ -94,21 +94,27 @@ export const PublishingStrategyOverview = () => {
 
             <Divider />
 
-            <Typography variant="h3">{t('editor.retention_strategy.rrs')}</Typography>
+            <Typography component="h2" variant="h3">
```

Comment:
Noen bestemt grunn til at vi vil stile denne annerledes enn en standard h2?
Ser dette gjelder flere plasser også, hvor jeg mistenker at vi egentlig vil fjerne overstyringen, sånn at en h2 som hovedregel faktisk ser ut som en h2

```suggestion
            <Typography variant="h2">
```